stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/src/main/java/com/epam/training/weapons/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.training.weapons;
+
+import com.epam.training.weapons.logic.DivisionLogic;
+import com.epam.training.weapons.model.AbstractWeapon;
+import com.epam.training.weapons.model.Blade;
+import com.epam.training.weapons.model.Division;
+import com.epam.training.weapons.view.DivisionPrinter;
+
+import static com.epam.training.weapons.model.Blade.HandCount.TWO_HAND;
+
+public class Main {
+
+  public static void main (String[] args){
+
+      DivisionCreator creator = new DivisionCreator();
+
+      Division division = creator.create();
+      AbstractWeapon blade = new Blade("Catana",1500,1000, TWO_HAND);
+      division.add(blade);
+      division.setName("Catana Division");
```

Comment:
better to pass name in constructor, in this case your division would be initialized with mandatory data in creation time